src/v/redpanda/application.cc

```diff
@@ -2132,14 +2132,26 @@ void application::wire_up_and_start_crypto_services() {
     construct_service(
       ossl_context_service,
       std::ref(*thread_worker),
-      ss::sstring{config::node_config().openssl_config_file().value_or("")},
-      ss::sstring{
```

Comment:
it should be a compiler error to call a zero-arg constructor with `()`. i will die on this hill.

Reply:
Would have saved me....